linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -271,7 +269,7 @@ MOCKABLE_FUNCTION(, int, threadpool_timer_start, THANDLE(THREADPOOL), threadpool
 
 **SRS_THREADPOOL_LINUX_07_057: [** `work_function_ctx` shall be allowed to be `NULL`. **]**
 
-**SRS_THREADPOOL_LINUX_07_058: [** `threadpool_timer_start` shall allocate a context for the timer being started and store `work_function` and `work_function_ctx` in it. **]**
+**SRS_THREADPOOL_LINUX_07_058: [** `threadpool_timer_start` shall allocate memory for `THANDLE_MALLOC(TIMER_INSTANCE)` and store `work_function` and `work_function_ctx` in it. **]**
```

Comment:
>`THANDLE_MALLOC(TIMER_INSTA [](http://example.com/codeflow?start=86&length=27)

We should probably add to the spec that we're passing threadpool_timer_dispose as dispose function

Reply:
added